packages/framework/src/Model/Order/Processing/OrderProcessorMiddleware/PersonalPickupPointMiddleware.php

```diff
@@ -51,7 +51,7 @@ public function handle(
                     $orderProcessingData->domainConfig->getId(),
                 );
 
-                $orderData->getItemsByType(OrderItem::TYPE_TRANSPORT)[0]->name .= ' ' . $store->getName();
+                // $orderData->getItemsByType(OrderItem::TYPE_TRANSPORT)[0]->name .= ' ' . $store->getName();
```

Comment:
this should be here or not? :slightly_smiling_face:

Reply:
That's the question 😆 
The original implementation contained this code, but it was never executed... So when I properly fix it and the personal pickup point is suffixed with the store name, the tests start failing...

Reply:
I have added the store name and fixed the tests